packages/jest-util/src/index.js

```diff
@@ -87,6 +87,19 @@ const warnAboutUnrecognizedOptions  = (argv: Object, options: Object) => {
   }
 };
 
+/**
+ * Normalizes relative paths to always use "/", even on Windows where "\" is the
```

Comment:
Again this should be in `scripts` or in the integration tests folder, not inside of jest-util. This isn't being used in Jest itself.